.spellcheckwordlist.txt

```diff
@@ -677,6 +678,7 @@ you
 your
 yoursite
 zcompdump
+zipfile
```

Comment:
Not a word unless we’re referring to Python’s [zipfile](https://docs.python.org/3/library/zipfile.html) library. (They write about “ZIP files” on that page!

```suggestion
```